modules/samples/artifacts/RDBMSSample/Store-rdbms.siddhi

```diff
@@ -14,10 +14,12 @@ Purpose:
 
 Prerequisites:
     1) Ensure that MySQL is installed on your machine.
-    2) Dowload the mysql jdbc driver and inser it the osgi bungled directory to
-        2.1) JDBC driver: https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.45.tar.gz
-        2.2) Extract it and use the "jartobundle" tool in {WSO2_SP_Home}/bin to convert the above jars into OSGi bundles.
-        2.3) Copy over the converted bundles to the {WSO2_SP_Home}/lib directory .
+    2) Download the mysql jdbc driver and insert it the osgi bungled directory to
```

Comment:
Replace this with
2) Add mysql jdbc driver into {WSO2_HOME}/lib
        2.1) Download JDBC driver (https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.45.tar.gz)